scripts/analytics/institution_summary.py

```diff
@@ -110,6 +110,10 @@ def get_class():
     init_app()
     institution_summary = InstitutionSummary()
     args = institution_summary.parse_args()
-    date = parse(args.date).date() if args.date else None
+    yesterday = args.yesterday
+    if not yesterday:
```

Comment:
Probably should be `if yesterday` and swap the conditions. That way if we wanted to put any more special conditions in, it would just be `elif`s.